docs/xblock-tutorial/anatomy/stylesheets.rst

```diff
@@ -0,0 +1,22 @@
+.. _The XBlock Stylesheets:
+
+######################
+The XBlock Stylesheets
+######################
+
+This section of the tutorial walks through the CSS file, `thumbs.css`_,  that
+is part of the Thumbs XBlock in the XBlock SDK.
+
+If you completed the steps in :ref:`Build an XBlock Quick Start`, you can find
+this file locally at ``xblock_development/xblock-sdk/sample_xblocks/thumbs/static/css/thumbs.css``.
```

Comment:
@feanil do you know why I get an error here?

```
ERROR: Unknown target name: "thumbs.css".
```

Reply:
I tried using a code directive but that didn't stop the error. Interestingly it's not making this error on `anatomy/javascript.rst`:
```
this file locally at ``xblock_development/xblock-sdk/sample_xblocks/thumbs/static/js/source/thumbs.js``.
```

Reply:
It's because of the link on Line 7 above, not because of this code block.